src/modules/timer/Timer.cpp

```diff
@@ -156,5 +156,11 @@ double Timer::getTime()
 #endif
 }
 
+double Timer::getTime()
+{
+	static const double start = getTimeAbsolute();
+	return getTimeAbsolute() - start;
```

Comment:
In the future I'll probably want to restructure this so `getTimeAbsolute` will return the platform-specific integer values directly, which would maximize precision of the final floating point getTime result - but that'd be a much more complicated change and this is a good change to have regardless, I think.

Reply:
I don't know if there is ever a need to use getAbsoluteTime (this is why I made it a static class function, because maybe there is). If you did it that way, you would not give that possibility anymore. But I don't think need the absolute version really.
After I un-refactored that conditional compilation stuff I thought it would be nice to have a single source file for each platform and just define getTime in there. And considering with different return types you need to wrap the whole functions (and definitions in the header) with #ifdef maybe it makes sense to just have getTime in the header and a separate source file for each platform? Then you can inline getTimerPeriod too and do `counter / frequency` directly instead of `counter * (1/frequency)`. I could also do that in the same file.